setup.py

```diff
@@ -44,7 +44,13 @@ def package_data(pkg, roots):
     entry_points={
         'xblock.v1': [
             'lti_consumer = lti_consumer:LtiConsumerXBlock',
-        ]
+        ],
+        'lms.djangoapp': [
+            "lti_consumer = lti_consumer:LTIConsumerApp",
+        ],
+        'cms.djangoapp': [
```

Comment:
Not sure if the Studio endpoint will be used, but adding it here in any case for now.